sources/packages/backend/libs/integrations/src/esdc-integration/e-cert-integration/e-cert-file-handler.ts

```diff
@@ -281,6 +281,8 @@ export class ECertFileHandler extends ESDCFileHandler {
       gender: application.student.gender,
       maritalStatus: application.relationshipStatus,
       studentNumber: application.studentNumber,
+      tuitionRemittanceEffectiveAmount:
```

Comment:
I believe e-cert file handler must not be modified(schoolAmount).  Only the assignment must change.